GUI/src/app/pages/main/left-menu/left-menu.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component } from '@angular/core';
+import {FormControl, ValidatorFn} from '@angular/forms';
+import { FormBuilder } from '@angular/forms';
+import { Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-left-menu',
+  templateUrl: './left-menu.component.html',
+  styleUrls: ['./left-menu.component.scss']
+})
+export class LeftMenuComponent {
+  constructor(private fb: FormBuilder) {
+    this.setDefaultInfoLeftMenu();
+  }
+  LeftMenuInfo = this.fb.group({
```

Comment:
Please, move this initialization to constructor, and this.setDefaultInfoLeftMenu(); to ngOnInit()